packages/generator-electrode/generators/webapp/templates/server/plugins/webapp/react-webapp.js

```diff
@@ -72,6 +72,14 @@ function makeRouteHandler(options, userContent) {
     const renderJs = RENDER_JS && mode !== "nojs";
     const renderSs = RENDER_SS && mode !== "noss";
 
+    if (!renderSs) {
+      if (options.request.server && options.request.server.app) {
+        options.request.app.disableSSR = true;
```

Comment:
inconsistent fields.  accessing `.app.disableSSR` but checking for `.server.app` above.